docs/source/noise/index.rst

```diff
@@ -114,51 +114,51 @@ Noise types for each column
     - Types of noise
     - Notes
   * - First name
-    - Decennial Census, ACS, CPS, WIC, W-2 and 1099, SSA
+    - Decennial Census, ACS, CPS, WIC, W-2 and 1099, 1040, SSA
     - Leave a field blank, use a fake name, nickname, make typos, make OCR errors, make phonetic errors
-    -
+    - In the 1040 form, the same noise types are applied to the corresponding columns for the joint filer and dependents
   * - Middle initial
-    - Decennial Census, ACS, CPS, WIC, W-2 and 1099, SSA
+    - Decennial Census, ACS, CPS, WIC, W-2 and 1099, 1040, SSA
     - Leave a field blank, make typos, make OCR errors, make phonetic errors
-    -
+    - In the 1040 form, this noise type functions the same for joint filer and dependent *n*
```

Comment:
```suggestion
    - In the 1040 form, the same noise types apply to the middle initial columns for the joint filer and dependents
```
As Zeb noted, there are several rows that need to be updated to be consistent. Here's an update that matches my suggestion for the first name column.